dashboards/mintel/cost-control/cost-analysis-dashboard.json

```diff
@@ -102,7 +102,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "sum (rate (container_cpu_usage_seconds_total{id=\"/\",service=\"kubelet\"}[1m])) / sum (machine_cpu_cores{service=\"kubelet\"}) * 100",
+          "expr": "sum (rate (container_cpu_usage_seconds_total{id!=\"/\",service=\"kubelet\"}[1m])) / sum (machine_cpu_cores{service=\"kubelet\"}) * 100",
```

Comment:
do we still need this `json` one since now we are rendering them ?

Reply:
I was leaving it in for comparison atm.